scripts/us_bjs/nps/nps_statvars.mcf

```diff
@@ -0,0 +1,979 @@
+Node: dcid:Count_Jurisdiction_Female
+gender: schema:Female
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
+Node: dcid:Count_Jurisdiction_WhiteAlone_Female
+race: dcs:WhiteAlone
+gender: schema:Female
+populationType: dcs:Person
+measurementQualifier: dcs:MeasuredBasedOnJurisdiction
+typeOf: dcs:StatisticalVariable
+institutionalization: dcs:Incarcerated
+statType: dcs:measuredValue
+measuredProperty: dcs:Count
+
```

Comment:
Generally, the StatVar should ~ follow

statType if not measuredValue + measuredProperty + populationType + values (in alphabetical property order) + mQual/other relevant stat var obs

so here, Count_Person_Female_Incacerated_WhiteAlone_MeasuredBasedOnJurisdiction